spearhead/header.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * The header for our theme
+ *
+ * This is the template that displays all of the <head> section and everything up until <div id="content">
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-files/#template-partials
+ *
+ * @package Seedlet
+ * @since 1.0.0
+ */
+?><!doctype html>
+<html <?php language_attributes(); ?>>
+<head>
+	<meta charset="<?php bloginfo( 'charset' ); ?>" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link rel="profile" href="https://gmpg.org/xfn/11" />
+	<?php wp_head(); ?>
+</head>
+
+<body <?php body_class(); ?>>
+<?php wp_body_open(); ?>
+<div id="page" class="site">
+	<a class="skip-link screen-reader-text" href="#content"><?php _e( 'Skip to content', 'seedlet' ); ?></a>
+
+		<header id="masthead" class="site-header default-max-width" role="banner">
+			<?php get_template_part( 'template-parts/header/site-branding' ); ?>
+
+			<?php if ( has_nav_menu( 'primary' ) ) : ?>
+				<nav id="site-navigation" class="primary-navigation" role="navigation" aria-label="<?php esc_attr_e( 'Main', 'seedlet' ); ?>">
+					<button id="primary-close-menu" class="button close">
+						<span class="dropdown-icon close"><?php _e( 'Close', 'seedlet' ); ?> <?php echo seedlet_get_icon_svg( 'close' ); ?></span>
+						<span class="hide-visually collapsed-text"><?php _e( 'collapsed', 'seedlet' ); ?></span>
+					</button>
+					<div class="extra-navigation-wrapper">
+						<?php
+						// Get menu slug
+						$location_name = 'primary';
+						$locations     = get_nav_menu_locations();
+						$menu_id       = $locations[ $location_name ];
+						$menu_obj      = wp_get_nav_menu_object( $menu_id );
+
+						wp_nav_menu(
+							array(
+								'theme_location'  => 'primary',
+								'menu_class'      => 'menu-wrapper',
+								'container_class' => 'primary-menu-container',
+								'items_wrap'      => '<ul id="%1$s" class="%2$s">%3$s</ul>',
+							)
+						);
+						?>
+						<?php if ( has_nav_menu( 'social' ) ) : ?>
+							<nav class="social-navigation" role="navigation" aria-label="<?php esc_attr_e( 'Social Links Menu', 'seedlet' ); ?>">
```

Comment:
Is it necessary or acceptable to add a `nav` element within a `nav` element? 

I tested out removing this extra nav element and adding the class `social-navigation` directly to the container generated by `wp_nav_menu`, and it seems to work about the same:

<img width="620" alt="Screen Shot 2020-10-28 at 11 59 51 AM" src="https://user-images.githubusercontent.com/5375500/97462660-58d08400-1915-11eb-9c6a-2db2fde0f021.png">

Reply:
good idea